checkov/secrets/git_history_store.py

```diff
@@ -37,7 +37,7 @@ def set_secret_map(self, file_results: List[PotentialSecret], file_name: str, co
             equal_secret_in_commit[secret_key].append(ADDED if secret.is_added else REMOVED)
 
         for secret in file_results:
-            if secret.filename in ['None', '']:
+            if secret.filename == '' or 'None' in secret.filename:
```

Comment:
Will catch `None` (the actual None) as well...
```suggestion
            if not secret.filename or 'None' in secret.filename:
```